test/helpers.js

```diff
@@ -5,7 +5,10 @@
 
 'use strict';
 
+var log = require('debug')('strong-runner:test:log');
+var err = require('debug')('strong-runner:test:err');
+
 exports.tapFriendlyConsole = {
-  log: console.error,
-  error: console.error,
+  log: log,
```

Comment:
I'll check, debug used to go to stderr, then it got changed to stdout, then they realized that was lame, and it went back to stderr. I think its still there.